src/component/Admin/Group/NewGroupDIalog.tsx

```diff
@@ -38,7 +38,7 @@ const defaultGroup: GroupEnt = {
     redirected_source: true,
   },
   edges: {
-    storage_policies: [],
+    storage_policies: undefined,
```

Comment:
Let's set this as `1` for new groups. Storage policy with ID=1 is protected and cannot be deleted, so it should always exist.